Overview.bs

```diff
@@ -464,27 +464,27 @@ not by wall-clock time, but by the progress of scrolling in a [=scroll container
 
     :   forwards
     ::  When the scroll offset is less than {{startScrollOffset}}, the
-        timeline's [=current time=] is 0.
-        When the scroll offset is greater than {{endScrollOffset}}, the
         timeline is inactive.
+        When the scroll offset is greater than {{endScrollOffset}}, the
```

Comment:
Instead of 'greater than', shouldn't this use ' greater than or equal' ? because when we have fill:forward the end point becomes inclusive. 

This apply to the rest of this patch as well.

Reply:
Done.